core/imageroot/usr/local/agent/actions/transfer-state/50sendstate

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+import subprocess
+import sys
+import json
+import agent
+import os
+
+request = json.load(sys.stdin)
+address = request['address']
+port = int(request['port'])
+replace = bool(request['replace'])
+username, password = request['credentials']
+
+errors = 0
+podman_cmd = [
+    'podman', 'run', '-i', '--workdir=/srv', '--rm', '--network=host', '--privileged',
+    '--env=RSYNC_PASSWORD=' + password,
+]
+
+rsync_cmd = [
+    'rsync',
+    '-a',
+    '--info=progress2',
+    '--delete-after',
+
+    # high-priority, harcoded filter rules: they cannot be overriden
+    '--exclude=/state/agent.env',
+    '--exclude=/state/environment',
+]
+
+if os.path.isfile('state-filter-rules.rsync'):
+    rsync_cmd += ['--filter=. state-filter-rules.rsync']
+
+rsync_cmd += [
+    './', # source: workdir
+    f'rsync://{username}@{address}:{port}/state/', # destination, remote state module
+]
+
+core_env = agent.read_envfile('/etc/nethserver/core.env') # Import URLs of core images
+
+transfer_cmd = [*podman_cmd, *agent.get_existing_volume_args(), core_env['RSYNC_IMAGE'], *rsync_cmd]
+print(*transfer_cmd, file=sys.stderr)
```

Comment:
It seems a debug statement.
For me it's fine to print it, but maybe we could add a `[INFO]` tag?

Reply:
Yes it's a bit debug, I can change it. 

We can hide it by putting a agent.SD_DEBUG log prefix. It might help to read the log and understand what is happening with `journalctl -p debug`. I think it's better than adding a string prefix, like [INFO] because that is not useful for full text searches.